### PR TITLE
updating installed extensions

### PR DESCRIPTION
**Pull request by cloin**
##### SUMMARY

This PR checks off some todo items for instruqt demos: https://github.com/ansible/instruqt/issues/48
- removes unsupported vscoss ansible extension
- adds WIP official ansible extension
- adds microsoft python to add some interpreter setting functionality within code-servver

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

- provisioner

### CODE REVIEW

File: roles/code_server/tasks/codeserver.yml [hunk 1]
```diff
@@ -73,7 +73,8 @@
   loop:
     - https://github.com/ansible/workshops/raw/devel/files/bierner.markdown-preview-github-styles-0.1.6.vsix
     - https://github.com/ansible/workshops/raw/devel/files/hnw.vscode-auto-open-markdown-preview-0.0.4.vsix
-    - https://github.com/ansible/workshops/raw/devel/files/vscoss.vscode-ansible-0.5.2.vsix
+    - https://github.com/ansible/workshops/raw/devel/files/redhat.ansible-0.4.5.vsix
```
[the remark this answers is not in the document]
Comment (webknjaz) [reply roles/code_server/tasks/codeserver.yml:76]: Also, should this PR remove the old file from the repo too?

**Comment by webknjaz** [comment]
Oh, and I've just noticed that your local Git setup is misconfigured. You probably forgot to do `git config --global user.email YOUR_GH_VERIFIED_ADDRESS`.

**Comment by cloin** [comment]
Recheck

**Comment by tonykay** [comment]
The F5 deployer appears to reference at least one of the old extensions @cloin 

`failed: [ce24-student1-ansible-1] (item=https://github.com/ansible/workshops/raw/devel/files/vscoss.vscode-ansible-0.5.2.vsix) => {"ansible_loop_var": "item", "changed": false, "dest": "/home/student1/.local/share/code-server/extensions/", "elapsed": 0, "gid": 1001, "group": "student1", "item": "https://github.com/ansible/workshops/raw/devel/files/vscoss.vscode-ansible-0.5.2.vsix", "mode": "0755", "msg": "Request failed", "owner": "student1", "response": "HTTP Error 404: Not Found", "secontext": "unconfined_u:object_r:data_home_t:s0", "size": 119, "state": "directory", "status_code": 404, "uid": 1001, "url": "https://github.com/ansible/workshops/raw/devel/files/vscoss.vscode-ansible-0.5.2.vsix"}`

**Pull request merged**